spec/features/routing/destinations/batch_update_spec.rb

```diff
@@ -163,6 +165,12 @@
       check :Short_calls_limit
       fill_in :short_calls_limit, with: assign_params[:short_calls_limit]
     end
+
+    if assign_params.key? :routing_tag_ids
+      check :Routing_tag_ids
+      routing_tags.select { |tag| assign_params[:routing_tag_ids].include? tag.id.to_s }
+                  .each { |tag| fill_in_chosen 'routing_tag_ids[]', with: tag.name, multiple: true, visible: false }
```

Comment:
```
page.scroll_to find_field('routing_tag_ids[]')
assign_params[:routing_tag_ids].each do |tag_id|
  fill_in_chosen 'routing_tag_ids[]', with RoutingTag.find(tag_id).name, multiple: true
end
```

it will be more readable - and no need for `visible: false` here